### DISCUSSION TITLE
Auth allowlist wildcards don't seem to work

### DISCUSSION THREAD

**Discussion by MaxTechnics** [supabase/supabase#28540] [Questions] [answered]
I've scrolled trough the discussions and the discord though i can't find my issue exactly. (I am aware there are a lot of similar issues, so i'm sorry if i missed an obvious thing somewhere)

I'm in the process of implementing a password reset flow, however the `redirectTo` parameter defaults to the site url. 

I have the following redirect url's in the allowlist:
![image](https://github.com/user-attachments/assets/747799cd-ab61-49af-830c-622fe9089483)

The code for requesting a password reset is as follows:
```js
const { error: supa_error } = await supabase.auth.resetPasswordForEmail(email.value, {
	redirectTo: `${window.location.hostname}/reset`
});
```
`window.location.hostname` resolves to https://[org].infonet.wtf or https://[org].infonet.media

I've tried a multitude of wildcards, none of which seemed to work, I used https://supabase.com/docs/guides/auth/redirect-urls as reference.

Any pointers would be more than welcome, since i'm at a loss right now.

Thanks in advance!

**Comment by j4w8n** [comment, marked as answer]
For the middle two redirects, you'll likely want to use a double * at the end.

For the last, I assume a single * for the subdomain should work, as they show in the example.

**Reply by MaxTechnics** [reply]
You're right. Even though the reset route can only be `domain.media/reset` Having the allowlist defined as such didn't seem to work.

`https://*.domain.com/**` Does seem to do the trick.

Thank you very much!!
